### PR TITLE
improve travis

### PR DESCRIPTION
**Pull request by danglotb**
Travis runs nopol on the bug 5 of chart from the [defects4j](https://github.com/rjust/defects4j) dataset in java7.

### CODE REVIEW

**Comment by monperrus** [comment]
does Travis  fail if the repair fails?

**Comment by danglotb** [comment]
He does not, yet. :)

**Comment by monperrus** [comment]
mergeable?

**Comment by danglotb** [comment]
It does. I test locally: when Nopol can not find any patch, travis fails.

File: .travis_run_tests.sh [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+    chmod +x ./.travis_run_defects4j.sh
+    ./.travis_run_defects4j.sh
+    if [[ $? == 1 ]]
+    then
+        exit 1
+    fi
+    echo "Running test suite... "
+    cd nopol
+    mvn clean verify jacoco:report coveralls:report
+    if [[ $? == 1 ]]
```
Comment (monperrus) [inline .travis_run_tests.sh]: != 0

**Review by monperrus — APPROVED** [review]

File: .travis_run_tests.sh [hunk 2]
```diff
@@ -1,4 +1,34 @@
...
+    then
+        exit 1
+    fi
+else
+    echo "Running test suite... "
+    cd nopol
+    mvn clean verify jacoco:report coveralls:report
+    if [[ $? == 1 ]]
+    then
```
Comment (monperrus) [inline .travis_run_tests.sh:31]: idem

**Pull request merged**